_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include post(9); };
+    }
+
+    .md-post-10 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include post(10); };
+    }
+
+    .md-post-11 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include post(11); };
+    }
+
+
+    // CASE: large screen bp
+    .lg-col-1 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(1); };
+    }
+
+    .lg-col-2 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(2); };
+    }
+
+    .lg-col-3 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(3); };
+    }
+
+    .lg-col-4 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(4); };
+    }
+
+    .lg-col-5 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(5); };
+    }
+
+    .lg-col-6 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(6); };
+    }
+
+    .lg-col-7 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(7); };
+    }
+
+    .lg-col-8 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(8); };
+    }
+
+    .lg-col-9 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(9); };
+    }
+
+    .lg-col-10 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(10); };
+    }
+
+    .lg-col-11 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(11); };
+    }
+
+    .lg-col-12 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include span(full); };
+    }
+
+    // shifts - pre
+    .lg-pre-1 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(1); };
+    }
+
+    .lg-pre-2 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(2); };
+    }
+
+    .lg-pre-3 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(3); };
+    }
+
+    .lg-pre-4 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(4); };
+    }
+
+    .lg-pre-5 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(5); };
+    }
+
+    .lg-pre-6 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(6); };
+    }
+
+    .lg-pre-7 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(7); };
+    }
+
+    .lg-pre-8 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(8); };
+    }
+
+    .lg-pre-9 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(9); };
+    }
+
+    .lg-pre-10 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(10); };
+    }
+
+    .lg-pre-11 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(11); };
+    }
+
+    // shifts - post
+    .lg-post-1 {
```

Comment:
NOTE: This whole set of definitions could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task once the grid gets some use.